src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -55,15 +56,14 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
   object f {
     import org.apache.mesos.{Protos => Mesos}
     val app = AppDefinition(id = PathId("/testapp"))
-    val runningMarathonInstance = TestInstanceBuilder.newBuilder(app.id, version = app.version, now = Timestamp.now()).addTaskRunning().getInstance()
```

Comment:
lots of changes here, but not necessarily related to LaunchEphemeral. Many tests in this actor did not make sense anymore as `TaskLauncherActor` does not compute it's internal state, instead it pull state all the time from instancetracker for every change.